CHANGELOG.md

```diff
@@ -5,6 +5,13 @@
 ### Removed
 
 - Dropped Node.js 7 support and require Node.js 8+
+- Dropped support for Nextcloud 9 and 10:
+  - owncloud tag in info.xml will not be migrated to nextcloud tags anymore
+  - ocsid will not be parsed anymore and was removed from the info.xsd
+  - old categories will not be migrated anymore (tool, game, productivity, other)
+  - presence of owncloud tag will not be validated anymore for apps depending on 9 and 10
```

Comment:
And of course they will be required to add a nextcloud tag now because we don't guess it from the owncloud one any more